src/engine/renderer/tr_shader.cpp

```diff
@@ -5971,6 +5963,14 @@ static shader_t *FinishShader()
 		}
 	}
 
+	if ( shader.sort <= Util::ordinal( shaderSort_t::SS_OPAQUE ) ) {
+		shader.fogPass = fogPass_t::FP_EQUAL;
+	} else if ( shader.contentFlags & CONTENTS_FOG ) {
+		shader.fogPass = fogPass_t::FP_LE;
+	}
+
+	shader.noFog = shader.noFog || shader.fogPass == fogPass_t::FP_NONE || ( backEnd.refdef.rdflags & RDF_NOWORLDMODEL );
```

Comment:
rdflags can change each frame so you can't use it here.

Reply:
Hmm, I thought it's only used in menus and such, but apparently it's only used for hyperspace.

Reply:
In fact, given that it's not used for anything else and hyperspace already has its own flag, I think `RDF_NOWORLDMODEL` can just be NUKED. Out of scope for this pr though.

Reply:
Anyway, fixed the initial issue.